src/renderer/features/multisigs/model/multisigs-model.ts

```diff
@@ -15,69 +12,71 @@ import {
   ExternalType,
   SigningType,
   WalletType,
+  Wallet,
 } from '@shared/core';
 import { networkModel, networkUtils } from '@entities/network';
 import { accountUtils, walletModel, walletUtils } from '@entities/wallet';
 import { MultisigResult, multisigService } from '@entities/multisig';
-import { multisigUtils } from '../lib/mulitisigs-utils';
 import { isEthereumAccountId, toAddress } from '@shared/lib/utils';
 import { CreateParams } from '@entities/wallet/model/wallet-model';
 import { notificationModel } from '@entities/notification';
+import { multisigUtils } from '../lib/mulitisigs-utils';
+
+const MULTISIG_DISCOVERY_TIMEOUT = 30000;
 
 const multisigsDiscoveryStarted = createEvent();
```

Comment:
I have kept the `multisigsDiscoveryStarted` event, because the `interval` method needs an event to `start`.